testgrid.tahoe-lafs.org/access-control.nix

```diff
@@ -0,0 +1,12 @@
+{ ... }: {
+  # Initial empty root password for easy login:
+  users.users.root.initialHashedPassword = "";
+  services.openssh.permitRootLogin = "prohibit-password";
+
+  services.openssh.enable = true;
+
+  users.users.root.openssh.authorizedKeys.keys = [
+    # Replace this by your SSH pubkey!
```

Comment:
Can we also document the process of getting the SSH pubkeys into the test grid machines?

I guess things like who should have access, and what they should and should not be doing, are a separate discussion, so maybe this can wait.